Objects/typeobject.c

```diff
@@ -57,8 +57,8 @@ static unsigned int next_version_tag = 0;
 
 typedef struct PySlot_Offset {
     int slot;
```

Comment:
`pyslot_offsets[i].slot` should always be `i`, so it should be possible to remove `slot`.

Reply:
You are right, have been updated, thanks.